File: app/src/main/java/org/nevack/unitconverter/model/HistoryItem.java
package org.nevack.unitconverter.model;

public class HistoryItem {
    private final int category;
    private final String valuefrom;
    private final String valueto;
    private final String unitfrom;
    private final String unitto;

    public HistoryItem(int category, String valuefrom, String valueto, String unitfrom, String unitto) {
        this.category = category;
        this.valuefrom = valuefrom;
        this.valueto = valueto;
        this.unitfrom = unitfrom;
        this.unitto = unitto;
    }

    public int getCategory() {
        return category;
    }

    public String getValuefrom() {
        return valuefrom;
    }

    public String getValueto() {
        return valueto;
    }

    public String getUnitfrom() {
        return unitfrom;
    }

    public String getUnitto() {
        return unitto;
    }
}

File: app/src/main/kotlin/dev/nevack/unitconverter/history/db/HistoryItem.kt
package dev.nevack.unitconverter.history.db

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "history")
data class HistoryItem(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    @ColumnInfo(name = "unit_from") val unitFrom: String,
    @ColumnInfo(name = "unit_to") val unitTo: String,
    @ColumnInfo(name = "value_from") val valueFrom: String,
    @ColumnInfo(name = "value_to") val valueTo: String,
    @ColumnInfo(name = "category") val category: Int,
)
